MAINTAINERS.md

```diff
@@ -22,6 +22,8 @@ another component.
 
 ### Lieutenant
 
+* [Jon Cowie](http://github.com/jonlives)
```

Comment:
If you're proposing LT, doesn't this also make you a maintainer by definition of the RFC?